src/services/meta_transaction_service.ts

```diff
@@ -228,7 +232,8 @@ export class MetaTransactionService {
             .callAsync();
 
         // log quote report and associate with txn hash if this is an RFQT firm quote
-        if (quoteReport) {
+        const shouldLogQuoteReport = quoteReport && params.apiKey !== undefined;
```

Comment:
i might be missing something: but I expected every meta txn request to always have an api key -- is that the case?